terraform/modules/forest_node/user-data.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+# Create a login user
+adduser --home "/home/${NEW_USER}" --disabled-password --gecos "" "${NEW_USER}"
+
+# Create .ssh directory for created user
+mkdir --parents -- "/home/${NEW_USER}/.ssh"
+chown "${NEW_USER}:${NEW_USER}" "/home/${NEW_USER}/.ssh"
+chmod 0700 "/home/${NEW_USER}/.ssh"
+
+# Check if authorized_keys file exists and copy it from root to new user
+if [ -f "/root/.ssh/authorized_keys" ]; then
+  cp /root/.ssh/authorized_keys "/home/${NEW_USER}/.ssh/authorized_keys"
+  chown "${NEW_USER}:${NEW_USER}" "/home/${NEW_USER}/.ssh/authorized_keys"
```

Comment:
```suggestion
if [ -f "/root/.ssh/authorized_keys" ]; then
  : Allowing those with root ssh keys to log in as ${NEW_USER}
  cp /root/.ssh/authorized_keys "/home/${NEW_USER}/.ssh/authorized_keys"
  chown "${NEW_USER}:${NEW_USER}" "/home/${NEW_USER}/.ssh/authorized_keys"
```
Why are we doing things this way - could you comment the file?
This is unusual to me, because if someone already has root access, then they could trivially access the `user` login. One explanation is that we always want to run forest as `user` instead of `root`.